app/components/ui/files/destinate-folder-modal/component.js

```diff
@@ -12,19 +13,19 @@ export default Ember.Component.extend({
   userAuth: service(),
   folderNavs: service(),
 
-  folders: Ember.A(), //Array of folders in the directory
+  folders: A(), //Array of folders in the directory
 
-  destinationFolder: Ember.Object.create({}),
-  previousFolder: Ember.Object.create({}),
-  selectionTree: Ember.Object.create({}),
-  directory: Ember.Object.create({}),
+  destinationFolder: Object.create({}),
+  previousFolder: Object.create({}),
+  selectionTree: Object.create({}),
+  directory: Object.create({}),
```

Comment:
This syntax seems bulky with no apparent gain in functionality. Nearly all of the methods on the classic `Object` are statically accessed, so I don't understand the value of using `Object.create({})` here when `{}` would effectively be the same and much more clear.

Am I missing something?

Reply:
The answer for this would be the same as above. By wrapping objects with  `EmberObject.create({})` you gain access to the properties within these objects and can add observers to those (if needed later) instead of having to go over a refactoring at a later stage.

Reply:
That makes sense.. While I disagree with the pattern of using them everywhere without an immediate need to observe any of the properties (e.g. `previousFolder`), the documentation on EmberObject made the benefits much more clear there.

Thank you!